mcs_benchmark_data/pipelines/commonsense_qa/commonsense_qa_benchmark_transformer.py

```diff
@@ -7,70 +7,76 @@
+from mcs_benchmark_data._benchmark_transformer import _BenchmarkTransformer
 from mcs_benchmark_data.models.benchmark import Benchmark
-from mcs_benchmark_data.models.benchmark_bootstrap import BenchmarkBootstrap
 from mcs_benchmark_data.models.benchmark_answer import BenchmarkAnswer
 from mcs_benchmark_data.models.benchmark_choice import BenchmarkChoice
 from mcs_benchmark_data.models.benchmark_concept import BenchmarkConcept
 from mcs_benchmark_data.models.benchmark_question import BenchmarkQuestion
 from mcs_benchmark_data.models.benchmark_question_type import BenchmarkQuestionType
 from mcs_benchmark_data.models.benchmark_sample import BenchmarkSample
-
-
-class CommonsenseQaBenchmarkTransformer(_Benchmark_Transformer):
-    def transform(self, **kwds) -> Generator[_Model, None, None]:
-        yield from _Benchmark_Transformer._transform(self, **kwds)
+from mcs_benchmark_data.pipelines.commonsense_qa.commonsense_qa_benchmark_file_names import (
+    CommonsenseQaBenchmarkFileNames,
+)
+
+
+class CommonsenseQaBenchmarkTransformer(_BenchmarkTransformer):
+    def transform(
+        self,
+        *,
+        extracted_path: Path,
+        file_names: CommonsenseQaBenchmarkFileNames,
+        **kwds,
+    ) -> Generator[_Model, None, None]:
+        yield from _BenchmarkTransformer._transform(
+            self, extracted_path=extracted_path, file_names=file_names, **kwds
+        )
 
     def _transform_benchmark_sample(
-        self, dataset_type: str, dataset_uri: URIRef, **kwds
+        self,
+        *,
+        extracted_path: Path,
+        file_names: CommonsenseQaBenchmarkFileNames,
+        dataset_type: str,
+        dataset_uri: URIRef,
+        **kwds,
     ) -> Generator[_Model, None, None]:
 
-        sample_jsonl_file_path = kwds["extracted_path"] / getattr(
-            kwds["file_names"], dataset_type + "_samples"
+        sample_jsonl_file_path = extracted_path / getattr(
+            file_names, dataset_type + "_samples"
         )
 
         with open(sample_jsonl_file_path) as file:
 
             all_samples = list(file)
 
-        ans_mapping = {ans: i for i, ans in enumerate("ABCDE")}
-
         for line in all_samples:
 
             sample = json.loads(line)
 
-            benchmark_sample_uri = URIRef(f"{dataset_uri}:sample:{sample['id']}")
-
             correct_choice = None
 
             if dataset_type != "test":
-                correct_choice = sample["answerKey"]
+                correct_choice = URIRef(
+                    f"{dataset_uri}:sample:{sample['id']}:correct_choice:{sample['answerKey']}"
+                )
 
-            yield from _Benchmark_Transformer._prepare_sample(
-                self,
-                dataset_type=dataset_type,
-                dataset_uri=dataset_uri,
-                sample=sample,
-                correct_choice=correct_choice,
-            )
+            concept = [sample["question"]["question_concept"]]
```

Comment:
Don't really need to assign all these variables if you're just going to pass them up to the superclass method.

Reply:
This should be a tuple, since that's what the superclass type expects.